### PR TITLE
queryable vcf file

### PR DESCRIPTION
**Pull request by MuhammedHasan**

### CODE REVIEW

**Comment by Hoeze** [comment]
@MuhammedHasan Does pybedtools vs. pyranges make any difference here?
+1 for merging this fast :grin:

**Comment by MuhammedHasan** [comment]
`pybedtools vs. pyranges` may cause breaking change in here.

File: kipoiseq/extractors/vcf_seq.py [hunk 1]
```diff
@@ -14,25 +22,339 @@
...
+        self.min_num = min_num
+
+    def __call__(self, variants, interval):
+        if self.max_num >= len(variants) >= self.min_num:
+            return [True] * len(variants)
+        else:
+            return [False] * len(variants)
+
+
+class VariantQueryable:
```
Comment (Avsecz) [inline kipoiseq/extractors/vcf_seq.py]: rename to `VariantIntervalQueryable`

File: kipoiseq/extractors/vcf_seq.py [hunk 2]
```diff
@@ -14,25 +22,339 @@
...
+
+    def __init__(self, vcf, variants, progress=False):
```
Comment (Avsecz) [inline kipoiseq/extractors/vcf_seq.py]: rename `variant` to `variant_interval`

**Pull request closed without merging**